lib/y2_module_basetest.pm

```diff
@@ -31,8 +31,23 @@ our @EXPORT = qw(is_network_manager_default
   continue_info_network_manager_default
   accept_warning_network_manager_default
   workaround_suppress_lvm_warnings
+  with_yast_env_variables
 );
 
+=head2 with_yast_env_variables
+
+ with_yast_env_variables([extra_vars]);
+
+Set environment variables for yast application.
+C<extra_vars> extends the variables that can be used. C<extra_vars> expects a string.
+ex: with_yast_env_variables("foo=bar");
+
+=cut
+sub with_yast_env_variables {
+    my ($self, $extra_vars) = shift // '';
```

Comment:
Sorry for asking the same as Martin already did, but what is the point to have $self in this function? You are not using it anywhere in the function:) 

If you'll remove it here, then you'll be able to call the function just as `with_yast_env_variables()` in `y2_module_consoletest.pm`. 

And it should resolve all the errors you've received in verification runs.

Reply:
@OleksandrOrlov i did some investigation. this is the deal
 y2_module_basetest.pm indeed exports the function but it also used as a superclass. Thus what we can do is to use only one of the two choices. export/import the function and call it as a function, or don't export it, use inheritance and call it via y2_module_basetest::with_yast_env_variables. I will go with the second.